spec/services/google_service/client_service_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GoogleService::ClientService, type: :service do
+  context 'when querying a simple keyword' do
+    it 'returns an HTTParty Response', vcr: 'google_search' do
```

Comment:
```suggestion
    it 'returns a HTTParty Response', vcr: 'google_search' do
```

Reply:
@junan  `an` vs `a` is decided based on sounding (and ease of pronunciation), not based on vowel/consonant. So "an HTTP request" or "an hour" are correct ;-)

I had this issue last month ;-)